.travis.yml

```diff
@@ -17,7 +17,7 @@ before_install:
   - ./travis-tool.sh bootstrap
 
 install:
-  - ./travis-tool.sh install_r RJSONIO
+  - ./travis-tool.sh install_r jsonlite
   - ./travis-tool.sh install_aptget r-cran-curl
```

Comment:
Sorry, that again.  Try adding `r-cran-jsonlite' on the second line, remove the first.  Will also make it faster (no compiles)